config/build.go

```diff
@@ -9,9 +9,9 @@ const (
 
 var (
 	// overridden by the build system
-	Version     = "dev"
-	BuildCommit = ""
-	BuildDate   = ""
+	BuildVersion = "dev"
```

Comment:
This will impact the replacement of `Version` from build tools. Those should be updated along with this change.
https://github.com/odpf/optimus/blob/main/Makefile#L17
https://github.com/odpf/optimus/blob/main/.goreleaser.yml#L13